pkg/u-boot/Dockerfile

```diff
@@ -8,7 +8,9 @@ SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
 ENV VERSION v2024.10-rc2
 ENV SOURCE_URL https://github.com/u-boot/u-boot/archive/${VERSION}.tar.gz
 ENV RASPBERRY_FIRMWARE_BLOBS_VERSION 1.20211007
+ENV RASPBERRY_FIRMWARE_BLOBS_VERSION2 1.20240306
```

Comment:
Should we use some more descriptive names, please?

Reply:
Probably, it would be better to get all the files from the newer version, but then I guess we should test on all raspberry devices to ensure that everything is ok.